src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkTask.java

```diff
@@ -107,6 +117,11 @@ public void flush(Map<TopicPartition, OffsetAndMetadata> offsets) {
   public void stop() {
     log.debug("Stopping Elasticsearch client.");
     client.close();
+    try {
+      highLevelClient.close();
```

Comment:
please set it to null also because in start we are doing `this.highLevelClient = highLevelClient != null ? highLevelClient : new RestHighLevelClient(`